container/src/main/java/com/flipkart/poseidon/Poseidon.java

```diff
@@ -227,6 +237,9 @@ private Handler getPoseidonHandler() {
         ServletContextHandler servletContextHandler = new ServletContextHandler();
         servletContextHandler.setContextPath("/");
         servletContextHandler.addServlet(new ServletHolder(getPoseidonServlet()), "/*");
+        servletContextHandler.addServlet(new ServletHolder(rotationCheckServlet), "/_poseidon/health");
```

Comment:
Let's call it rotation instead of health in url?

For health, we can use existing __metrics/ping url.

Reply:
done